### PR TITLE
refactor(directives): [click-outside] use JSX in Unit test

### PR DESCRIPTION
**Pull request by holazz**
#6114 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/directives/__tests__/click-outside.test.tsx [hunk 1]
```diff
@@ -112,33 +87,29 @@ describe('Directives.vue', () => {
...
-    const wrapper = mount(OuterComponent, {
-      global: {
-        directives: {
-          ClickOutside,
+    const wrapper = mount(
+      {
+        setup() {
+          return () => (
+            <div class="other-class" v-click-outside={secondHandler}>
+              <div class="trigger" v-click-outside={firstHandler}>
```
Comment (ryuhangyeong) [inline packages/directives/__tests__/click-outside.test.tsx:98]: ## Suggestion

I think it's better to use a **constant** that has already been defined. `OTHER_CLASS`, `TRIGGER`

File: packages/directives/__tests__/click-outside.test.tsx [hunk 2]
```diff
@@ -151,7 +122,7 @@ describe('Multiple click-outside directives', () => {
     expect(firstHandler).toHaveBeenCalledTimes(1)
     expect(secondHandler).toHaveBeenCalledTimes(1)
 
-    // update the component instance
+    // // update the component instance
```
Comment (ryuhangyeong) [inline packages/directives/__tests__/click-outside.test.tsx:125]: ## Etc

I think the comment was misspelled!

**Review by ryuhangyeong — APPROVED** [review]

File: packages/directives/__tests__/click-outside.test.tsx (hunk 1 above)
Comment (holazz) [reply packages/directives/__tests__/click-outside.test.tsx:98]: Good catch!

**Comment by holazz** [comment]
updated.

**Comment by ryuhangyeong** [comment]
Thank you 🙇

**Pull request merged**